app/src/main/java/org/eu/exodus_privacy/exodusprivacy/fragments/apps/AppsFragment.kt

```diff
@@ -79,7 +80,21 @@ class AppsFragment : Fragment(R.layout.fragment_apps) {
                 binding.shimmerLayout.visibility = View.GONE
                 appsRVAdapter.submitList(it)
             } else {
+                binding.swipeRefreshLayout.visibility = View.VISIBLE
                 binding.shimmerLayout.visibility = View.VISIBLE
+                Snackbar.make(
```

Comment:
Ok, i removed the snackbar code. It got re-added when pulling. 
It might be just enough to have the swipeRefreshLayout visible to be able to refresh at any time.